server/service/src/processors/general_processor.rs

```diff
@@ -1,12 +1,18 @@
 use async_trait::async_trait;
+use log::info;
 use repository::{
-    ChangelogFilter, ChangelogRepository, ChangelogRow, ChangelogTableName, EqualFilter, KeyType,
-    RepositoryError, TransactionError,
+    ChangelogFilter, ChangelogRepository, ChangelogRow, ChangelogTableName, EqualFilter,
+    PluginType, RepositoryError, TransactionError,
 };
+use strum::Display;
 use thiserror::Error;
 
 use crate::{
-    cursor_controller::CursorController,
+    backend_plugin::{
+        plugin_provider::{PluginError, PluginInstance},
```

Comment:
quite a few changes in this file, main change, is being able to processor multiple processor for one type (for plugins), this needed a vec to be returned

Reply:
Probably best to view diff ignoring whitespace